### ISSUE TITLE
there is a bug from torchgeo.datasets import ETCI2021

### ISSUE THREAD

**Issue by Yeah21** [microsoft/torchgeo#933]
### Description
```
Traceback (most recent call last):
  File "/home/wbz/Works/ETCI2021FloodDetection/main.py", line 3, in <module>
    from torchgeo.datasets import ETCI2021
  File "/home/wbz/anaconda3/envs/torch/lib/python3.6/site-packages/torchgeo/datasets/__init__.py", line 11, in <module>
    from .chesapeake import (
  File "/home/wbz/anaconda3/envs/torch/lib/python3.6/site-packages/torchgeo/datasets/chesapeake.py", line 311, in <module>
    class ChesapeakeCVPR(GeoDataset):
  File "/home/wbz/anaconda3/envs/torch/lib/python3.6/site-packages/torchgeo/datasets/chesapeake.py", line 347, in ChesapeakeCVPR
    crs = CRS.from_epsg(3857)
  File "/home/wbz/anaconda3/envs/torch/lib/python3.6/site-packages/rasterio/crs.py", line 333, in from_epsg
    obj._crs = _CRS.from_epsg(code)
  File "rasterio/_crs.pyx", line 332, in rasterio._crs._CRS.from_epsg
rasterio.errors.CRSError: The EPSG code is unknown. PROJ: proj_create_from_database: /home/wbz/anaconda3/envs/torch/lib/python3.6/site-packages/rasterio/proj_data/proj.db contains DATABASE.LAYOUT.VERSION.MINOR = 0 whereas a number >= 2 is expected. It comes from another PROJ installation.
free(): invalid pointer

Process finished with exit code 134
```

### Steps to reproduce
```python
from torchgeo.datasets import ETCI2021
```
### Version

0.2.1

**Comment by calebrob6** [comment]
Hi @Yeah21, this is a problem with conflicting package versions (e.g. I usually see this happen when you mix conda and pip installs of rasterio/fiona). Can you try creating a new virtual environment, then `pip install torchgeo`?

**Comment by adamjstewart** [comment]
Is there a reason you're using TorchGeo 0.2.1? 0.3.1 will contain a lot of new features and bug fixes, although it removes Python 3.6 support.

**Comment by Yeah21** [comment]
the question has been solved， thanks！
